### DISCUSSION TITLE
Palette stuffs

### DISCUSSION THREAD

**Discussion by MRMIdAS** [Phobos-developers/Phobos#877] [Ideas and feedback]
Is it possible to have custom palettes behave like other buildings? maybe a tag? like "darkens=" which makes the game think its the same as a regular building SHP and it just shifts down the palette when a lightning storm happens?

**Comment by GraionDilach** [comment]
Related Ares ticket/discussion: https://bugs.launchpad.net/ares/+bug/1024367
